hardware_interface/src/resource_manager.cpp

```diff
@@ -540,6 +663,58 @@ class ResourceStorage
     import_command_interfaces(systems_.back());
   }
 
+  void add_sub_controller_manager(
```

Comment:
Does resource manager have to know about controller manager? It's a bit convoluted.